sources/personio/README.md

```diff
@@ -1,48 +1,59 @@
----
-title: Personio
-description: dlt source for Personio API
-keywords: [personio api, personio source, personio]
----
-
-
 # Personio
 
-[Personio](https://personio.de/) is a holistic HR software for companies from 10 - 2000 employees.
-It is one of the relevant data sources for our organisation.
+[Personio](https://personio.de/) is a human resources management software that helps businesses 
+streamline HRprocesses, including recruitment, employee data management, and payroll, in one 
+platform.
 
 Resources that can be loaded using this verified source are:
 
-| Name        | Description                                                                                                                  |
-| ----------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| employees   | List Company Employees using the [employees](https://api.personio.de/v1/company/employees) endpoint.                         |
-| absences    | Provides a list of absence types using the [time-off-types](https://api.personio.de/v1/company/time-off-types) endpoint.     |
-| attendances | Fetch attendance data for the company employees using [attendances](https://api.personio.de/v1/company/attendances) endpoint.|
+| Name        | Description                                                                              |
+|-------------|------------------------------------------------------------------------------------------|
+| employees   | Retrieves company employees details. (employees_list, absense_entitlement, cost_centers) |
+| absences    | Retrieves list of various types of employee absences                                     |
+| attendances | Retrieves attendance records for each employee                                           |
+
+The `employees` endpoint includes the following details:
+
+| Name                | Description                                                                                 |
+|---------------------|---------------------------------------------------------------------------------------------|
+| employees_list      | A directory of all employees with their details like names, titles, and contact information |
+| absence_entitlement | Records of each employee's leave rights, including vacation and sick days                   |
+| cost_centers        | Organizational units for tracking and allocating employee-related costs                     |
```

Comment:
I meant explain it to me. they are child tables? we don't need to describe them here, because if you set max_nesting_level=0, they won't appear.
I would delete them from readme, they are not resources

Reply:
The `employees` endpoint includes the following details:

| Name                | Description                                                                                 |
|---------------------|---------------------------------------------------------------------------------------------|
| employees_list      | A directory of all employees with their details like names, titles, and contact information |
| absence_entitlement | Records of each employee's leave rights, including vacation and sick days                   |
| cost_centers        | Organizational units for tracking and allocating employee-related costs                     |